Homework_8.py

```diff
@@ -0,0 +1,73 @@
+import json  # import module for work with json
+from Homework_7 import *  # import previous homeworks (5-6)
```

Comment:
It's time to add some more structure to the code, not just import previous modules which importing more early modules and so on, let's create a module for Publishers, module for normalization, module for counting words/letters, module for common scripts like define_format etc.